hack/install-kcp-edge.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 --create-folder --verbose
+
+# This script installs KCP-Edge binaries to a folder of choice
+#
+# Arguments:
```

Comment:
I myself also like to document the dependency/significance if any, or document the lack thereof if that is the case, on the current working directory --- because as user I know that sometimes this matters.